backend/src/main/java/reviewme/highlight/service/dto/HighlightsRequest.java

```diff
@@ -0,0 +1,11 @@
+package reviewme.highlight.service.dto;
+
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+public record HighlightsRequest(
+
+        @NotNull(message = "하이라이트할 부분을 입력해주세요.")
+        List<HighlightRequest> highlights
```

Comment:
validation annotation 이 붙은 컬렉션의 각 원소에도 valid 검증을 하기 위해서는 여기에됴 @Valid 붙여줘야 합니다!
```suggestion
        @NotNull(message = "하이라이트할 부분을 입력해주세요.") @Valid
        List<HighlightRequest> highlights
```

지금은 아래 요청이 통과되네요

![image](https://github.com/user-attachments/assets/816c3dc5-1a17-4d48-8cbc-9d17e2c832db)

Reply:
헉 그럼 다른데에도 붙여줘야겠네요? 확인해보니 속에는 안 붙인데가 있는데
예: ReviewRegisterRequest 속의 ReviewAnswerRequest를 위해서 붙여야함

Reply:
네네 그래서 이번에 싹 다 붙여도 될 것 같아용~